src/domain/profile/Profile.tsx

```diff
@@ -16,18 +15,13 @@ import styles from './profile.module.scss';
 const Profile: FunctionComponent = () => {
   const { loading, error, data } = useQuery<ProfileQueryType>(profileQuery);
   const { t } = useTranslation();
-  const history = useHistory();
   const dispatch = useDispatch();
   let profile;
 
-  if (loading)
-    return (
-      <div>
-        <LoadingSpinner isLoading={true} />
-      </div>
-    );
+  if (loading) return <LoadingSpinner isLoading={true} />;
   if (error || !get(data, 'guardians.edges[0]') || !data) {
-    history.push('/home');
+    // Temporarily comment out cause of invalid state update
```

Comment:
Maybe a TODO comment would be good so it is easy to find this one

Reply:
added